third_party/move/move-command-line-common/src/values.rs

```diff
@@ -60,7 +60,7 @@ pub enum ParsedValue<Extra: ParsableValue = ()> {
     Custom(Extra),
 }
 
-pub trait ParsableValue: Sized {
+pub trait ParsableValue: Sized + Sync + Send + Clone + 'static {
```

Comment:
How come these extra trait bounds are necessary?

Reply:
There's a bunch of weird explicit inputs are necessary now.  The developers working on it moved from a larger codebase and implicit inputs to a smaller codebase and explicit inputs.  Inputs now need to be clonable, and sometimes lifetimes need to be cleaned up / in this case it needs to be okay to send multithreaded